css/global.css

```diff
@@ -31,6 +31,16 @@
   ):last-child {
   display: none !important;
 }
+
+/* hides applicant count on linkedin.com/jobs in row with location */
+[class*="job-details-jobs-unified-top-card"]
```

Comment:
[last-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child) hides the last child in a list of sibling elements. Since the element is heavily nested and the class name was changed (perhaps to deemphasize the applicant count altogether), I made these changes.